package.json

```diff
@@ -30,6 +30,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.8.4",
+    "@babel/eslint-parser": "^7.16.5",
```

Comment:
I think adding new npm package should have been done in separate PR. The reason is not to pollute this PR with those changes.